src/store/modules/order/actions.ts

```diff
@@ -104,8 +104,7 @@ const actions: ActionTree<OrderState , RootState> ={
     } else if(orderType === 'packed') {
       payload['shipmentStatusId']= "SHIPMENT_PACKED"
     } else {
-      dispatch('updateCurrent', { order: {} })
-      return;
+      payload['shipmentStatusId']= "ITEM_COMPLETED"
```

Comment:
Update the condition for fetching completed orders, you can check the payload prepared for the fetching completed orders